From c9059cc51e26a56c14b02cc5462940df59dd5373 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=3D=3FUTF-8=3Fq=3FJouni=3D20K=3D2E=3D20Sepp=3DC3=3DA4nen?=
 =?UTF-8?q?=3F=3D?= <jks@iki.fi>
Date: Mon, 5 Oct 2015 07:35:34 +0300
Subject: Fix encoding for LaTeX

Instead of a combining acute accent, use a single-codepoint encoding
Patch-Name: 9bc518230684b3a8bb69af60e160f90e1091cd0c.patch
---
 doc/style_changes.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/style_changes.rst b/doc/style_changes.rst
index 15da551..6a8e0e0 100644
--- a/doc/style_changes.rst
+++ b/doc/style_changes.rst
@@ -10,7 +10,7 @@ upcoming 2.0 release!
 The new default color map will be 'viridis' (aka `option
 D <http://bids.github.io/colormap/>`_).  For an introduction to color
 theory and how 'viridis' was generated watch Nathaniel Smith and
-Stéfan van der Walt's talk from SciPy2015
+Stéfan van der Walt's talk from SciPy2015
 
 .. raw:: html
 
